uploads/logos/.gitignore

```diff
@@ -0,0 +1 @@
+*.png
```

Comment:
Did you already made an argument for storing this locally an not in mongo?
I know that mongo isn't ideal, but it probably makes backups a lot easier (and developing with a mirror).